test/integration/dualstack/dualstack_endpoints_test.go

```diff
@@ -253,7 +253,7 @@ func TestDualStackEndpoints(t *testing.T) {
 			// wait until the endpoint slices are created
 			err = wait.PollImmediate(1*time.Second, wait.ForeverTestTimeout, func() (bool, error) {
 				lSelector := discovery.LabelServiceName + "=" + svc.Name
-				esList, err := client.DiscoveryV1beta1().EndpointSlices(ns.Name).List(context.TODO(), metav1.ListOptions{LabelSelector: lSelector})
+				esList, err := client.DiscoveryV1().EndpointSlices(ns.Name).List(context.TODO(), metav1.ListOptions{LabelSelector: lSelector})
```

Comment:
Line 27, the import still references the beta 
```
test/integration/dualstack/dualstack_endpoints_test.go:27:      discovery "k8s.io/api/discovery/v1beta1"
```

Reply:
Thanks for catching that!